lua/neo-tree/sources/filesystem/init.lua

```diff
@@ -41,6 +41,8 @@ local follow_internal = function(callback, force_show, async)
   end
   if not state.path then
     return false
+  else
+    state.path = utils.normalize_path(state.path)
```

Comment:
This fix should be in `M._navigate_internal` before the path is first set rather than here. Probably at line 125 just before

``` lua
if path ~= state.path then
```

should be:

``` lua
path = utils.normalize_path(state.path)
if path ~= state.path then
```

Reply:
fixed, please review again. 😄